.env-example

```diff
@@ -1,9 +0,0 @@
-MAILTRAP_USER=InboxUsername
-MAILTRAP_PASSWORD=InboxPassword
-
-# Camo proxies insecure images in students' status updates
-CAMO_HOST=https://rgsoc-teams-camo.herokuapp.com
-CAMO_KEY=supersecret
-
-# Google Maps API key for e.g. city autocomplete
-GOOGLE_MAPS_API_KEY=YourGoogleMapsAPIKey
```

Comment:
We don't want to remove this file from the repo. (I don't think you need to remove it locally, either 🤔 ).
Anyway, removing it should not be part of this PR.

Reply:
yes,sorry for that,the file got accidentally removed ,will add it back now :)